packages/kit/src/runtime/server/page/crypto.js

```diff
@@ -0,0 +1,239 @@
+const encoder = new TextEncoder();
```

Comment:
should we add something at the top of this file like:
```
// TODO: remove this in favor of web crypto API once we no longer support Node 14
```

Reply:
yep, done